src/App.tsx

```diff
@@ -480,6 +481,7 @@ function App(): ReactElement {
       }
 
       canv.setFeature(newFeatureName);
+      return newFeatureName;
```

Comment:
`newFeatureName` is an argument to this function and is never modified within its body. Could the caller just use the value they passed in rather than getting it back out of this function?

Reply:
There's an early return statement that returns the existing `featureName` state if the new feature could not be found.

Reply:
Makes sense!